### ISSUE TITLE
Deprecate and/or remove the demand_scaling parameter to the model building function

### ISSUE THREAD

**Issue by danielolsen** [Breakthrough-Energy/REISE.jl#144]
# :rocket:

- [ ] Is your feature request essential for your project?

### Describe the workflow you want to enable
No change in user-facing functionality.

### Describe your proposed implementation
Remove this parameter and any associated logic, since it relates to old logic (i.e. reduce demand by 5% after each infeasible interval) which is no longer used, now that we have per-bus and per-time-period load shedding.

**Comment by BainanXia** [comment]
Yeah, this old logic for Matpower should only live in REISE instead of REISE.jl.

**Comment by merrielle** [comment]
Low priority but very easy
